associacao-brasileira-de-normas-tecnicas-ufrgs.csl

```diff
@@ -309,14 +373,25 @@
           </group>
         </group>
       </else-if>
+      <else-if type="article-newspaper" match="any">
+        <group>
+          <text variable="volume" prefix="ano " suffix=", "/>
+          <text variable="edition" prefix="ano " suffix=", "/>
```

Comment:
this seems weird and might be a misunderstanding of "edition" -- the way the term is understood in CSL, it refers to a specific version of the same work. For newspapers this might be the evening edition or a local edition of a national paper. I don't think you want to preface it with "ano"

Reply:
Done. Thanks!

Reply:
You still have edition prefaced with "ano" -- you can say if that's correct, but please don't resolve discussions unless they're actually resolved (or just let me resolve them). It makes reviewing much harder for me.

Reply:
Resolvido. Obrigada. E desculpa por ter resolvido a conversa, estava na dúvida de como você saberia que tinha respondido.